gh_actions_exporter/config.py

```diff
@@ -35,6 +38,8 @@ class ConfigFile(BaseSettings):
 
 
 class Settings(BaseSettings):
+    load_dotenv()
```

Comment:
For support of `.env` files use the method documented by the libray: https://docs.pydantic.dev/latest/usage/settings/#dotenv-env-support

Reply:
Ok, I'm working on it !

Reply:
Done